polynote-spark/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -204,7 +204,7 @@ object SocketTransport {
     ): IO[DeployedProcess] = {
       val sparkConfig = config.spark ++ notebookConfig.sparkConfig.getOrElse(Map.empty)
 
-      val sparkArgs = (sparkConfig - "sparkSubmitArgs" - "spark.driver.extraJavaOptions" - "spark.submit.deployMode")
+      val sparkArgs = (sparkConfig - "sparkSubmitArgs" - "spark.driver.extraJavaOptions" - "spark.submit.deployMode" - "spark.driver.memory")
```

Comment:
btw, I couldn't find `sparkSubmitArgs` in the normal spark config docs - is it something you are setting in your local `config.yml`? If so, maybe we should add it to the template?

Reply:
It's not a real config item 😞 it's just a fake "magic" one that I use to be able to pass args directly to spark-submit by editing an item in the spark params.